data/abixen-platform/misuses/3/misuse.yml

```diff
@@ -0,0 +1,15 @@
+api:
+- javax.crypto.Cipher
+violations:
+- insecure/condition/transformation
+description: >
+  First parameter in Cipher.getInstance(String) is with value "AES/CBC/PKCS5PADDING" which should be any of AES/CBC/{Empty String, PKCS7Padding, PKCS5Padding, ISO10126Padding}.
```

Comment:
Is the problem here the incorrect casing of the padding? If so, please state explicitly.